packages/forms/src/widgets/MultiSelectTagWidget/MultiSelectTagWidget.scss

```diff
@@ -34,9 +34,14 @@ $tf-typeahead-section-border-color: #e0e0e0 !default;
 		> div {
 			width: 100%;
 		}
+
+		&[class*="container-open"] .items-container {
```

Comment:
doesn't `&.container-open .items-container` work here ?

Reply:
css modules generates unique ending for classname. your solution exists in typeahead styles and works everywhere except IE. that's why i added this workaround to not touch existing workable code